ci-operator/step-registry/hypershift/ibmcloud/cluster-claim/hypershift-ibmcloud-cluster-claim-chain.yaml

```diff
@@ -9,7 +9,7 @@ chain:
       if [[ "${BUILD_ID}" != "" ]]; then
         echo "Build ${BUILD_ID} installed the current HO deployment."
       fi
-      exit 1
+      exit 0
```

Comment:
I think as written this would cause the rest of the steps in the chain to be run, but we want it to exit early. Perhaps there's some goofy restructuring of steps & pre-steps that we could use to create the different control flow ... but maybe we need changes in the OCP test platform itself to support early exit.

Another option would be to write a sentinel file to the $SHARED_DIR that steps can all access, and each subsequent step can look for that sentinel and, if present, they exit early as well.

Reply:
@stevekuznetsov something like https://github.com/openshift/release/pull/48608/commits/4fd0d5a556f4a00e4d5a0416c1b089cc3f6f7dc4 perhaps?

Reply:
That looks good, but perhaps `exit 0` or we will still be reporting failure in that case, yes?

Reply:
In the follow-up checks, I think we'd need to `exit 0` as well, or those failed steps will fail the job as well.

Reply:
oh oh oh, I got what you're saying. I'll push that up now

Reply:
updated in https://github.com/openshift/release/pull/48608/commits/30798353901210747cc2aaf1000e1c6b713a75d2